opencxl/cxl/device/pci_to_pci_bridge_device.py

```diff
@@ -0,0 +1,131 @@
+"""
+ Copyright (c) 2024, Eeum, Inc.
+
+ This software is licensed under the terms of the Revised BSD License.
+ See LICENSE for details.
+"""
+
+from dataclasses import dataclass
+from asyncio import Queue, create_task, gather
+
+from opencxl.util.logger import logger
+from opencxl.util.component import RunnableComponent
+from opencxl.cxl.component.cxl_connection import CxlConnection
+
+
+@dataclass
+class BindPair:
+    source: Queue
+    destination: Queue
+
+
+class BindProcessor(RunnableComponent):
```

Comment:
1. Make this the base class (does not belong in this file once it becomes a base class).
2. Create `class VppbBindProcessor` (replace `BindProcessor` in opencxl/cxl/component/virtual_switch/port_binder.py) and `class PpbBindProcessor` for PPB.
3. Reorganize file locations so that the two make sense.

Reply:
I took the liberty of cleaning it up in a different way that better suits how we've currently implemented bindings:

- Created `opencxl/cxl/component/bind_processor.py`.
- Base, private class: `_GenericBindProcessor` for commonizing.
- Inherited to `PpbDspBindProcessor` for PPB-DSP binding.
- Inherited to `VppbPpbBindProcessor` for vPPB-PPB binding, this class adds `_vcs_id`, `_vppb_id` members that's needed for this specific relation.

We expect `PpbDspBindProcessor` to be further customized for upcoming MLD changes.